packages/interactive-messages/package.json

```diff
@@ -63,6 +64,10 @@
     "tsscmp": "^1.0.6"
   },
   "devDependencies": {
+    "@microsoft/api-extractor": "^7.3.4",
+    "@types/debug": "^4.1.4",
```

Comment:
not sure why this package got new devDependencies besides `@microsoft/api-extractor`, when most of the other packages did not.

Reply:
also, we haven't changed convention to moving `@types/*` dependencies into `devDependencies` (discussion in #830)

Reply:
🤔 Good question. I think this is the result of a bad merge commit (read: my fault while merging).